news/2025-07-31-new-accelerate-macos.md

```diff
@@ -0,0 +1,16 @@
+# New Accelerate support for macOS 13.3+
+
+conda-forge by default uses `OpenBLAS` as its BLAS and LAPACK
+provider on macOS as it was updated regularly and is the least
+buggiest performant BLAS/LAPACK implementation.
+
+macOS 13.3 updated the Accelerate framework after a long time with
+improved support for LAPACK APIs and has long-time known bugs in
```

Comment:
```suggestion
improved support for LAPACK APIs and has long-time known bugs in
```

Do we have any links on this info for those who want to check the bugs?

Reply:
I meant to say bug fixes, not bugs. :smile: